lms/djangoapps/courseware/tests/test_video_handlers.py

```diff
@@ -256,7 +256,7 @@ def test_available_translation_en(self):
 
         request = Request.blank('/available_translations')
         response = self.item.transcript(request=request, dispatch='available_translations')
-        self.assertEqual(json.loads(response.body), ['en'])
+        self.assertEqual(json.loads(response.body.decode('utf-8')), ['en'])
```

Comment:
can we use simplejson here?

Reply:
@Ayub-Khan Too many jsons operations going on in this file. So its better to simply use decode here.